base_exception/migrations/10.0.2.0.0/pre-migration.py

```diff
@@ -8,5 +8,11 @@
 @openupgrade.migrate(use_env=True)
 def migrate(env, version):
     cr = env.cr
-    openupgrade.rename_tables(cr, [('sale_exception', 'exception_rule')])
-    openupgrade.rename_models(cr, [('sale.exception', 'exception.rule')])
+    
```

Comment:
Remove the space ` ` in order to fix
`base_exception/migrations/10.0.2.0.0/pre-migration.py:11:1: W293 blank line contains whitespace`

Same case for
`base_exception/migrations/10.0.2.0.0/pre-migration.py:15:1: W293 blank line contains whitespace`

please